modules/ppcp-onboarding/src/Render/OnboardingOptionsRenderer.php

```diff
@@ -68,8 +66,10 @@ public function render( bool $is_shop_supports_dcc ): string {
 		<label><input type="checkbox" id="ppcp-onboarding-accept-cards" ' . $checked . '> ' . __( 'Securely accept all major credit & debit cards on the strength of the PayPal network', 'woocommerce-paypal-payments' ) . '</label>
 	</li>
 	<li>' . $this->render_dcc( $is_shop_supports_dcc ) . '</li>' .
-			$this->render_pui_option()
-		. '</ul>';
+			$this->render_pui_option();
+		return ' <ul class="ppcp-onboarding-options">' .
+			apply_filters( 'ppcp_onboarding_options', $on_boarding_options );
+		'</ul>';
```

Comment:
I think it should be a  dot `.` after `apply_filters(...)` instead of a semicolumn `;`